modules/flowable-idm-engine/src/test/java/org/flowable/idm/engine/test/api/identity/GroupQueryTest.java

```diff
@@ -179,150 +165,130 @@ public void testQueryByMember() {
 
         query = query.orderByGroupId().asc();
         List<Group> groups = query.list();
-        assertEquals(3, groups.size());
-        assertEquals("admin", groups.get(0).getId());
-        assertEquals("frogs", groups.get(1).getId());
-        assertEquals("muppets", groups.get(2).getId());
+        assertThat(groups).hasSize(3);
+        assertThat(groups.get(0).getId()).isEqualTo("admin");
+        assertThat(groups.get(1).getId()).isEqualTo("frogs");
+        assertThat(groups.get(2).getId()).isEqualTo("muppets");
```

Comment:
Can we replace constructs like this (testing order of elements) with something like:

```
assertThat(groups).containsExactly("admin", "frogs", "muppets");
```